app/MyApplicationLoader.scala

```diff
@@ -27,30 +32,37 @@ class MyComponents(context: Context)
   extends BuiltInComponentsFromContext(context)
     with AhcWSComponents
     with CSRFComponents
+    with HttpConfigurationComponents
+    with AssetsComponents
```

Comment:
Theoretically it could have been left as deprecated but working, but for some reason the assets controller running in backwards compatibility mode always 404ed locally.   rather than trying to diagnose it, it was quicker to actually update to DI.